tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/constant/SimpleConstantNegativeTest.java

```diff
@@ -117,7 +117,7 @@ public void testNegative() {
         BAssertUtil.validateError(compileResult, index++, "incompatible types: expected 'IntTypeWithoutType', " +
                 "found 'int'", 183, 28);
         BAssertUtil.validateError(compileResult, index++, "incompatible types: expected 'ByteTypeWithType', " +
-                "found 'int'", 194, 26);
+                "found 'byte'", 194, 26);
```

Comment:
I'm not sure if this correct. Types of numeric literals at the moment (without singleton types) are int, float, or decimal.

Reply:
Updated. 424163fb2d7a81b614d6440d153fe0035b34b6e3